deployer/auth.py

```diff
@@ -43,13 +43,17 @@ def get_connections(self):
             for connection in self.auth0.connections.all()
         }
 
-    def create_client(self, name, domains):
+    def create_client(self, name, domains, connection_name):
         callbacks = self._get_callback_url_list(domains)
+        logout_urls = []
+        if connection_name == 'password':
```

Comment:
I noticed that if we define this `allowed_logout_url` list even if we're not using a password auth, it doesn't cause any harm. So, these checks aren't necessarily needed. I was thinking to remove them. What do you think?

Reply:
Yeah, doing it unconditionally is probably not a bad idea - we want everyone to log out regardless of connection type. Right now I think if you log out of our hub and log out of github as well, if you try to log in again you'll just immediately log in - no need to go via github. Making this unconditional should fix this.